namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-intelligence-dev/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: manage-intelligence-dev
+spec:
+  limits:
+  - default:
+      cpu: 1000m
+      memory: 1024Mi
+    defaultRequest:
+      cpu: 10m
+      memory: 1024Mi
```

Comment:
Please revert this defaultRequest memory value back to the default of 100Mi

For more information about these limits, see [this article](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/concepts/namespace-limits.html#namespace-container-resource-limits)